### ISSUE TITLE
[49.19.1] Artillery Counter Battery Null pointer exception

### ISSUE THREAD

**Issue by DeniedExistence31** [MegaMek/megamek#5585]
### Environment
Megamek/MekHQ Version: 0.49.19.1 milestone
OS: Windows 10
Java Version: Adoptium 11.0.23.9

### Description
Princess encounters a null point exception and freezes with artillery bot while counter-battery fire is going on against offboard artillery. Restarting and kicking/replacing the bot do not resolve the issue.

### Files
Game save
[buggedarty.sav.gz](https://github.com/user-attachments/files/15862106/buggedarty.sav.gz)
Log files
[megamek_artynullpointerexception.log](https://github.com/user-attachments/files/15862112/megamek_artynullpointerexception.log)

**Comment by repligator** [comment]
I get the same issue when loading the save file on 49.19.1. I tried playing _as_ the Tikonov Artillery just to see if anything changed. Everything runs fine unless the offboard artillery is targeted. The same NPE seems to occur, but the log entry is longer. Not sure if it's significant at all; but I've attached them anyway.
[ai.megamek.log](https://github.com/user-attachments/files/15878401/ai.megamek.log)
[human.megamek.log](https://github.com/user-attachments/files/15878402/human.megamek.log)
